extensions/mssql/snippets/mssql.json

```diff
@@ -226,7 +226,7 @@
 		"body": [
 			"-- Get a list of tables and views in the current database",
 			"SELECT table_catalog [database], table_schema [schema], table_name name, table_type type",
-			"FROM information_schema.tables",
+			"FROM INFORMATION_SCHEMA.TABLES",
 			"GO"
 		],
 		"description": "List tables and vies in the current database"
```

Comment:
Side note: since you're in this code, could you fix the typo `vies` -> `views`? If not I'll fix internally
'